### PR TITLE
HYDRA-934 : Fix reload of a scene with a Usd stage.

### PR DESCRIPTION
**Pull request by lanierd-adsk**

### CODE REVIEW

**Comment by lanierd-adsk** [comment]
FYI : @debloip-adsk @lilike-adsk @vlasovi @roopavr-adsk

**Comment by lanierd-adsk** [comment]
@ppt-adsk  : unfortunately, I could not find a way to repro the problem with a unit test, the test always passes with image comparison (a C++ would be the same as the prims are present and visible).

File: lib/mayaHydra/mayaPlugin/renderOverride.cpp [hunk 1]
```diff
@@ -1218,6 +1218,10 @@ void MtohRenderOverride::_InitHydraResources(const MHWRender::MDrawContext& draw
     //Add the scene index as an input scene index of the merging scene index
     _renderIndexProxy->InsertSceneIndex(_mayaHydraSceneIndex, SdfPath::AbsoluteRootPath());
     
+    if (!_sceneIndexRegistry) {
```
Comment (lanierd-adsk) [inline lib/mayaHydra/mayaPlugin/renderOverride.cpp:1221]: This change is not really related to the bug but I think it's better to pull out this from the _CreateSceneIndicesChain function as it will be ignored most of the time.

**Review by ppt-adsk — APPROVED** [review]
